src/Packages/PHPJava/Extended/_Object.php

```diff
@@ -41,12 +41,9 @@ public function __call($name, $arguments)
         throw new NoSuchMethodException($name . ' does not exist on ' . get_class($this));
     }
 
-    /**
-     * @return _Object
-     */
-    public function __default_clone(): _Object
+    public function __default_clone(): void
```

Comment:
Hey! Do not change return type because PHP cannot override it.

Reply:
Hmm, I don't think `__default__clone` will be overriden by any methods. Any opinions so far?

Reply:
Sorry, I saw wrong as "__clone" 🙇‍♀ 
Seem to tired.